sdk/python/flet/animated_switcher.py

```diff
@@ -112,9 +113,9 @@ def __init__(
         #
         duration: Optional[int] = None,
         reverse_duration: Optional[int] = None,
-        switch_in_curve: Optional[AnimationCurve] = None,
-        switch_out_curve: Optional[AnimationCurve] = None,
-        transition: Optional[AnimatedSwitcherTransition] = None,
+        switch_in_curve: Union[AnimationCurve, AnimationCurveString] = None,
```

Comment:
It's not an optional anymore?

Reply:
If you want I can change it to:  `Union[AnimationCurve, AnimationCurveString, None]`
But then, I already added an Optional at the level of the function to deal with that..
```python
@beartype
    def __set_switch_in_curve(self, value: Optional[AnimationCurveString]):
```
What should I do please?

Reply:
Yep, I think `Union[AnimationCurve, AnimationCurveString, None]` is the way to go.

Otherwise it shows an error:

![image](https://user-images.githubusercontent.com/5041459/206822442-32082b5a-a46d-4503-9828-ca649b2ed960.png)

Reply:
Check other similar cases too if there are any.

Reply:
or just add `None` to `AnimationCurveString`?

Reply:
added `None` to `Union`